### PR TITLE
Use internal folder by Standard Go Project Layout

### PR DESCRIPTION
**Pull request by t25kim**
Signed-off-by: Taewan Kim <t25.kim@samsung.com>

# Description

This PR is intended to refactor folder structures by Standard Go Project Layout.

Related #209

## Type of change

- [x] Code cleanup/refactoring
- [x] Documentation update

# How Has This Been Tested?
Build and Test Suite in Actions

**Test Configuration**:
* Edge Orchestration Release: Coconut

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by t25kim** [comment]
[2 Security Hotspots](https://sonarcloud.io/project/security_hotspots?id=lf-edge_edge-home-orchestration-go&pullRequest=272&resolved=false&types=SECURITY_HOTSPOT) has been detected from SonarCloud.
I'd like to resolve them as a new issue since they've appeared from old code.

File: internal/controller/storagemgr/storagedriver/tests.log [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+level=ERROR ts=2021-01-22T00:56:01.587816248Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading Random is out of the value type(21)'s range"
+level=ERROR ts=2021-01-22T00:58:04.286154926Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading -123.junk is out of the value type(19)'s range"
+level=ERROR ts=2021-01-22T00:58:04.291884501Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading Random is out of the value type(21)'s range"
+level=ERROR ts=2021-01-22T02:58:06.063788028Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading -123.junk is out of the value type(19)'s range"
+level=ERROR ts=2021-01-22T02:58:06.065130197Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading Random is out of the value type(21)'s range"
+level=ERROR ts=2021-01-25T02:50:32.205478144Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading -123.junk is out of the value type(19)'s range"
+level=ERROR ts=2021-01-25T02:50:32.206797153Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading Random is out of the value type(21)'s range"
```
Comment (tdrozdovsky) [inline internal/controller/storagemgr/storagedriver/tests.log:34]: Please remove the `internal/controller/storagemgr/storagedriver/tests.log` file from the tracking

**Review by tdrozdovsky — CHANGES_REQUESTED** [review]

File: internal/controller/storagemgr/storagedriver/tests.log (hunk 1 above)
Comment (t25kim) [reply internal/controller/storagemgr/storagedriver/tests.log]: @tdrozdovsky Thank you very much for reviewing carefully and not merging unnecessary file!

**Review by tdrozdovsky — APPROVED** [review]
LGTM!

**Comment by t25kim** [comment]
@MoonkiHong Code conflicts were fixed.

**Comment by MoonkiHong** [comment]
Security hotspot (from SonarCloud analysis) will be dealt by the registered issue #273.

**Review by MoonkiHong — APPROVED** [review]
LGTM. Thank you!!

**Review by suresh-lc — APPROVED** [review]
For MNEDC related issue, we are working on it and hence would be resolved as part of #273

**Comment by tdrozdovsky** [comment]
I think it is necessary to merge this PR with the master branch!

**Pull request merged**